### PR TITLE
select interaction files are added

### PR DESCRIPTION
**Pull request by deepa-thomas**
- [ ] changing color of answer divs when user selects an answer
- [ ] makes all other divs unclickable

### CODE REVIEW

File: client/src/presentation/listeners/select-listener.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { changeColor } from '../handlers/select-handler.js';
+
+// changing the color of answers div for correct and incorrect answers
+document.querySelectorAll('.div-answer').forEach((element) => {
+  element.addEventListener('click', changeColor);
```
Comment (katsmamina) [inline client/src/presentation/listeners/select-listener.js:5]: after having a look ...I thought that it's better to add class `disable` to all divs, including the clicked one. it just would look more natural

File: client/styles/index.css [hunk 2]
```diff
@@ -137,7 +142,7 @@ h2 {
 
 .answer {
   background-color: #decba4;
-  padding: 0.5rem;
+  padding: 0.2rem;
```
Comment (katsmamina) [inline client/styles/index.css:140]: made a smaller border, otherwise the div becomes too big

File: client/src/business-logic/get-answer.js [hunk 3]
```diff
@@ -4,8 +4,6 @@ export const accessAnswers = (i = 0) => {
   const data = findAll();
   const questionsObject = data[0];
   const questionsArray = Object.values(questionsObject);
-  console.log(questionsArray[1][0].answers[1].text);
```
Comment (katsmamina) [inline client/src/business-logic/get-answer.js]: I think in the future we should use `find.js` function to choose the right id (just thoughts)

**Review by katsmamina — COMMENTED** [review]
there is another thing to improve for the future: 
after we click an answer-div, and before we remove the mouse, the wide turquoise border remains present. 
 
Maybe it should be done within another user-story, just a comment for now.

**Comment by katsmamina** [comment]
@deepa-thomas check at the small changes I made, if you approve, it can be merged.

File: client/src/presentation/listeners/select-listener.js (hunk 1 above)
Comment (deepa-thomas) [reply client/src/presentation/listeners/select-listener.js:5]: Yes you are right.

File: client/src/business-logic/get-answer.js (hunk 3 above)
Comment (danielhalasz) [reply client/src/business-logic/get-answer.js]: yeah, I was trying to use find(), instead of findAll(), but could not figure out how to make it work

**Review by danielhalasz — COMMENTED** [review]
just a small remark..for the first question, multiple answers are correct..which makes everything a bit more complex..should we for now stick to a more simple scenario, where there is only 1 correct answer for each question?

**Pull request merged**